\hypertarget{classpip_1_1__vendor_1_1urllib3_1_1exceptions_1_1TimeoutError}{}\doxysection{pip.\+\_\+vendor.\+urllib3.\+exceptions.\+Timeout\+Error Class Reference}
\label{classpip_1_1__vendor_1_1urllib3_1_1exceptions_1_1TimeoutError}\index{pip.\_vendor.urllib3.exceptions.TimeoutError@{pip.\_vendor.urllib3.exceptions.TimeoutError}}


Inheritance diagram for pip.\+\_\+vendor.\+urllib3.\+exceptions.\+Timeout\+Error\+:\nopagebreak
\begin{figure}[H]
\begin{center}
\leavevmode
\includegraphics[width=350pt]{classpip_1_1__vendor_1_1urllib3_1_1exceptions_1_1TimeoutError__inherit__graph}
\end{center}
\end{figure}


Collaboration diagram for pip.\+\_\+vendor.\+urllib3.\+exceptions.\+Timeout\+Error\+:\nopagebreak
\begin{figure}[H]
\begin{center}
\leavevmode
\includegraphics[width=246pt]{classpip_1_1__vendor_1_1urllib3_1_1exceptions_1_1TimeoutError__coll__graph}
\end{center}
\end{figure}


\doxysubsection{Detailed Description}
\begin{DoxyVerb}Raised when a socket timeout error occurs.

Catching this error will catch both :exc:`ReadTimeoutErrors
<ReadTimeoutError>` and :exc:`ConnectTimeoutErrors <ConnectTimeoutError>`.
\end{DoxyVerb}
 

The documentation for this class was generated from the following file\+:\begin{DoxyCompactItemize}
\item 
inventaria\+\_\+env/lib/python3.\+10/site-\/packages/pip/\+\_\+vendor/urllib3/exceptions.\+py\end{DoxyCompactItemize}
